test/test_suite.py

```diff
@@ -198,3 +199,81 @@ class TableDDLTest(_TableDDLTest):
     )
     def test_underscore_names(self):
         pass
+
+
+class StringTest(_StringTest):
+    @provide_metadata
+    def _literal_round_trip(self, type_, input_, output, filter_=None):
+        """
+        SPANNER OVERRIDE:
+
+        Spanner is not able cleanup data and drop the table correctly,
+        table was already exists after related tests finished, so it doesn't
+        create a new table and when started new tests, following
+        insertions will fail with `400 Duplicate name in schema: t.
+        Overriding the tests to create a new table for test and drop table manually
+        before it creates a new table to avoid the same failures.
+        """
```

Comment:
minor nit:
```
        SPANNER OVERRIDE:
        Spanner is not cleaning up data and dropping the table correctly,
        as the DDL statements are being queued, so it doesn't
        create a new table and when started new tests, following
        insertions will fail with `400 Duplicate name in schema: t.
        Overriding the tests to create a new table for test and drop table manually
        before it creates a new table to avoid the same failures.
```

Please add a TODO to remove this override as mentioned in the other PR.

Reply:
Disregard